atomic_reactor/utils/operator.py

```diff
@@ -431,12 +431,27 @@ def _named_pullspecs(self):
         return pullspecs
 
     def _related_image_pullspecs(self):
+        """Get the pullspecs from spec.relatedImages section
+
+        :return: a list of pullspecs. It could be an empty if no spec.RelatedImage section.
+        :rtype: list[RelatedImage]
+        """
         related_images_path = ("spec", "relatedImages")
         return [
             RelatedImage(r)
             for r in chain_get(self.data, related_images_path, default=[])
         ]
 
+    def get_related_image_pullspecs(self):
```

Comment:
Could we also have a simple unit test for this in `test_operator.py`?

Reply:
Unit test is added.